src/pages/taskHistory/ColumnHeaders.test.tsx

```diff
@@ -32,115 +36,144 @@ const initialState: HistoryTableReducerState = {
 interface WrapperProps {
   children: React.ReactNode;
   state?: Partial<HistoryTableReducerState>;
+  mocks?: MockedProviderProps["mocks"];
 }
 
-const wrapper: React.FC<WrapperProps> = ({ children, state }) => (
-  <HistoryTableProvider initialState={{ ...initialState, ...state }}>
-    {children}
-  </HistoryTableProvider>
+const wrapper: React.FC<WrapperProps> = ({ children, state, mocks }) => (
```

Comment:
can this be replaced by making a modification to ProviderWrapper in HistoryTable/hooks/test-utils.tsx

Reply:
Great Idea!